allennlp/data/dataset_readers/masked_language_modeling.py

```diff
@@ -5,8 +5,8 @@
 
 from allennlp.data.instance import Instance
 from allennlp.data.tokenizers.tokenizer import Tokenizer
-from allennlp.data.tokenizers import Token, WordTokenizer
-from allennlp.data.tokenizers.word_splitter import JustSpacesWordSplitter
+from allennlp.data.tokenizers import Token
+from allennlp.data.tokenizers.word_tokenizer import JustSpacesTokenizer
```

Comment:
Can we call this `WhitespaceTokenizer` instead?  And it looks like these tokenizers should be moved to separate files, instead of keeping them all in `word_tokenizer`.